extensions/dacpac/package.nls.json

```diff
@@ -0,0 +1,4 @@
+{
+	"dacFx.settings": "Dacpac",
+	"dacFx.defaultSaveLocation": "Full path to folder where dacpacs and bacpacs are saved by default"
```

Comment:
Maybe "where .DACPAC and .BACPAC files are saved by default" to make it clear that those are file types.